manic/repository_svn.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, component_name, repo, ignore_ancestry=False):
     # Public API, defined by Repository
     #
     # ----------------------------------------------------------------
-    def checkout(self, base_dir_path, repo_dir_name, verbosity):
+    def checkout(self, base_dir_path, repo_dir_name, verbosity, recursive):  # pylint: disable=unused-argument
```

Comment:
Can you add a comment here that the 'recursive' argument is ignored, and is just here for compatibility with the general API?

Reply:
Done.